tests/utils.js

```diff
@@ -24,6 +24,7 @@ const userSettings = require('./factories/cht/users/user-settings');
 let originalSettings;
 const originalTranslations = {};
 let e2eDebug;
+const hasModal = async () => await element(by.css('#update-available')).isPresent();
```

Comment:
You don't need async + await here. returning the promise is enough: 
```suggestion
const hasModal = () => element(by.css('#update-available')).isPresent();
```